src/status_im/contexts/wallet/events/collectibles.cljs

```diff
@@ -0,0 +1,127 @@
+(ns status-im.contexts.wallet.events.collectibles
+  (:require [camel-snake-kebab.core :as csk]
+            [camel-snake-kebab.extras :as cske]
+            [clojure.string :as string]
+            [re-frame.core :as rf]
+            [taoensso.timbre :as log]
+            [utils.ethereum.chain :as chain]
+            [utils.transforms :as types]))
+
+(def collectibles-request-batch-size 1000)
+
+(defn displayable-collectible?
+  [collectible]
+  (let [{:keys [image-url animation-url]} (:collectible-data collectible)]
+    (or (not (string/blank? animation-url))
+        (not (string/blank? image-url)))))
```

Comment:
`empty?` is a better alternative here.

Reply:
@smohamedjavid Why do you say `empty?` is better?

There are a few rasons why I think `blank?` is better:

If we use `empty?` it would be:
```clojure
(or (not (empty? animation-url))
    (not (empty? image-url)))
```
   and as `empty?` docs say, prefer using `(seq x)` instead of `not` + `empty?`.

Another reason is `empty?` works also on sequences, so it might be harder to know what the vars are (seqs or strings), so using `string/blank` let us know we want to check strings.